jpa/subsystem/src/main/java/org/jboss/as/jpa/container/TransactionScopedEntityManager.java

```diff
@@ -62,18 +63,20 @@
     private final String puScopedName;          // Scoped name of the persistent unit
     private final Map properties;
     private transient EntityManagerFactory emf;
+    private transient EntityManagerCache cache;
```

Comment:
This should probably be final.

Reply:
Thanks, made this also final, need to be transient so we don't serialize the cache itself with the entity manager